libraries/monitor/src/monitor.js

```diff
@@ -131,6 +131,8 @@ class Monitor {
 
           this.log.expressTimer({
             name,
+            hasAuthed: req.hasAuthed,
+            clientId: req.hasAuthed ? await req.clientId() : '',
```

Comment:
neat, so this will avoid logging `auth-failed` for things that didn't need auth..

Reply:
I think it is also required because an endpoint with no scopes will need to talk to auth to do `req.clientId()` whereas it is cheap to do for an already authed endpoint.